### ISSUE TITLE
Re bug-hunter-init-file, emacs -q -l is not exactly identical to emacs

### ISSUE THREAD

**Issue by thblt** [Bruce-Connor/elisp-bug-hunter#23]
I'm facing a bug which happens only if the configuration is loaded the regular way, but *not* with `emacs -q -l ~/.emacs.d/init.el`.  The details are not really important, but this specific property makes `bug-hunter-init-file`  unusable, since even loading the full file with `-q -l init.el` doesn't trigger the bug.

The following screenshot shows the difference between the two invocations.  Left is `emacs -q -l ~/.emacs.d/init.el temp/test.org`, right is `emacs  temp/test.org`. 
 
![2018-06-23-191002_3840x2160_scrot](https://user-images.githubusercontent.com/2453136/41812049-91961fd2-771b-11e8-8170-a6c6850ff3e4.png)

I'm not sure yet what the root cause is, but I suspect elisp-bug-hunter would be closer from "regular" emacs invocation if it did offer the option to override $HOME instead of using `-q -l`

thanks!

**Comment by Malabarba** [comment]
> I'm not sure yet what the root cause is, but I suspect elisp-bug-hunter would be closer from "regular" emacs invocation if it did offer the option to override $HOME instead of using -q -l

Yes, I think you're right. But I think it would probably be harder to make it cross-plaform.
Ultimately, bug-hunter is never going to be able to catch every single corner case. It's primarily designed as an aid for newer users with the most common issues.
